docs/flink_processor.md

```diff
@@ -6,12 +6,27 @@ sections we describe how to use the Flink processor.
 ## Deployment Mode
 The Flink processor runs the Flink job in one of the following mode:
 
+- None
 - Session Mode
 - Kubernetes Application Mode
 
 Session mode is the default mode. User can specify the deployment mode with 
 configuration `deployment_mode`.
 
+### None
```

Comment:
Maybe rename this mode as `CLI Mode` for consistency with the `./flink run` described here[1]?

https://nightlies.apache.org/flink/flink-docs-master/docs/deployment/cli/#cli-actions

BTW, do you think we need to add `CLI Application Mode` corresponding to `./flink run-application` in the future?

Reply:
I don't think we will add `CLI Application Mode` in the future. Feathub doesn't need to know whether it runs with application or session modes. It only needs to know that the user will submit the Feathub job with Flink CLI.